.zenodo.json

```diff
@@ -1,65 +1,62 @@
-  "references": [
-    "Emmert, J. T., A. D. Richmond, and D. P. Drob (2010), A computationally compact representation of Magnetic-Apex and Quasi-Dipole coordinates with smooth base vectors, J. Geophys. Res., 115(A8), A08322, doi:10.1029/2010JA015326.",
-    "Richmond, A. D. (1995), Ionospheric Electrodynamics Using Magnetic Apex Coordinates, Journal of geomagnetism and geoelectricity, 47(2), 191–212, doi:10.5636/jgg.47.191."
-  ],
-  "keywords": [
-    "Magnetic Apex Coordinates",
-    "Quasi Dipole Coordinates",
-    "MLT",
-    "Magnetic Local Time",
-    "Conversion",
-    "Coordinate Conversion",
-    "Converting",
-    "Ionosphere",
-    "Magnetic Field",
-    "Space Physics",
-    "Heliophysics"
-  ],
   "creators": [
     {
-      "orcid": "0000-0002-8043-0953",
-      "name": "van der Meeren, Christer"
+      "name": "van der Meeren, Christer",
+      "orcid": "0000-0002-8043-0953"
     },
     {
-      "orcid": "0000-0001-5028-4943",
-      "name": "Laundal, Karl M."
+      "name": "Laundal, Karl M.",
+      "orcid": "0000-0001-5028-4943"
     },
     {
-      "orcid": "0000-0001-8875-9326",
       "affiliation": "Naval Research Laboratory",
-      "name": "Burrell, Angeline G."
+      "name": "Burrell, Angeline G.",
+      "orcid": "0000-0001-8875-9326"
     },
     {
-      "orcid": "0000-0001-7098-0524",
-      "name": "Lamarche, Leslie L."
+      "name": "Lamarche, Leslie L.",
+      "orcid": "0000-0001-7098-0524"
     },
     {
-      "orcid": "0000-0002-3487-3630",
-      "name": "Starr, Gregory"
+      "name": "Starr, Gregory",
+      "orcid": "0000-0002-3487-3630"
     },
     {
-      "orcid": "0000-0002-4621-3453",
-      "name": "Reimer, Ashton S."
+      "name": "Reimer, Ashton S.",
+      "orcid": "0000-0002-4621-3453"
     },
     {
-      "orcid": "0000-0001-7955-4441",
       "affiliation": "GFZ German Research Centre for Geosciences",
-      "name": "Morschhauser, Achim"
+      "name": "Morschhauser, Achim",
+      "orcid": "0000-0001-7955-4441"
     },
     {
-      "orcid": "0000-0001-9741-4063",
       "affiliation": "GFZ German Research Centre for Geosciences",
-      "name": "Michaelis, Ingo"
+      "name": "Michaelis, Ingo",
+      "orcid": "0000-0001-9741-4063"
     },
     {
-      "orcid": "0000-0001-8321-6074",
       "affiliation": "Goddard Space Flight Center",
-      "name": "Klenzing, Jeff"
+      "name": "Klenzing, Jeff",
+      "orcid": "0000-0001-8321-6074"
     }
-  ]
+  ],
+  "keywords": [
+    "Magnetic Apex Coordinates",
+    "Quasi Dipole Coordinates",
+    "MLT",
+    "Magnetic Local Time",
+    "Conversion",
+    "Coordinate Conversion",
+    "Converting",
+    "Ionosphere",
+    "Magnetic Field",
+    "Space Physics",
+    "Heliophysics"
+  ],
+  "license": {
+    "id": "MIT"
+  },
+  "title": "apexpy",
+  "version": "2.1.0"
```

Comment:
Questions:
- Why is the version included?
- Why are the notes and references missing?

Reply:
Hmm. This is the autogenerated version, so the version number was just copied. I'll look into it.